backend/battling/tests/test_forms.py

```diff
@@ -105,3 +110,69 @@ def test_create_team_form_with_invalid_pokemon(self):
         )
 
         self.assertFalse(form.is_valid())
+
+
+class PokemonApiIntegrationTest(TestCaseUtils):
+    def setUp(self):
+        super().setUp()
+        self.opponent = baker.make("users.User")
+        self.battle = baker.make("battling.Battle", creator=self.user, opponent=self.opponent)
+        self.team = baker.make("battling.Team", battle=self.battle, trainer=self.user)
+
+    @patch("pokemon.helpers.get_pokemon_from_api")
+    def test_if_mock_works_on_pokemon_api_integration(self, mock_get_pokemon):
```

Comment:
This name is confusing :eyes: A better name would `test_create_team_form_calls_get_pokemon_from_api`

Reply:
The name of this test is still confusing